suit_generator/suit/manifest.py

```diff
@@ -125,6 +125,18 @@ class SuitIndex(SuitUnion):
     )
 
 
+class SuitSequenceNumber(SuitUint):
+    """Representation of unsigned int type."""
```

Comment:
nit: Consider something like: `Representation of SUIT sequence number`

Reply:
Fixed!